packages/Button/src/IconButton.styles.js

```diff
@@ -25,21 +26,21 @@ const iconButtonSizes = {
   }`,
 };
 
-const minorStyles = `
+const minorStyles = css`
   transition: background-color 0.2s ease-out;
-  
+
   &:hover {
-    background-color: ${stylers.alpha(tokens.color.black, 0.1)}; 
+    background-color: ${stylers.alpha(tokens.color.black, 0.1)};
```

Comment:
no action need it but we should decide soon if we want to use: { alpha } or stylers.alpha ...